pages/linux/upower.md

```diff
@@ -0,0 +1,24 @@
+# upower
+
+> System utility to provide battery information and statistics.
```

Comment:
I think we should change this too. Either:
```suggestion
> System utility to provide power and battery information and statistics.
```
or 
```suggestion
> System utility to provide power information and statistics.
```

Reply:
Commited option 1